lisa/sut_orchestrator/azure/transformers.py

```diff
@@ -77,6 +77,8 @@ class VhdTransformerSchema(schema.Transformer):
     storage_account_name: str = ""
     container_name: str = DEFAULT_EXPORTED_VHD_CONTAINER_NAME
     file_name_part: str = ""
+    # Users can define the blob full name
+    custom_blob_full_name: str = ""
```

Comment:
Give an example of the value. When is it needed? Can it use the same value as vhd?

Reply:
Added.